### PR TITLE
[WIP] Fixes #5910 - Realgud layer

### LINKED ISSUE #5910: Feature request: support realgud
Would it be possible/reasonable to add support for realgud?

https://www.emacswiki.org/emacs/DebuggingWithEmacs#toc3

https://github.com/rocky/emacs-dbgr

I guess it's a better debugging front-end for interacting with external debuggers than built-in(?) GUD.

### PR DESCRIPTION
**Pull request by jluttine**
Add realgud layer. See issue #5910 for more details.

This is not working yet. I made a pull request so the code can be commented easily.

### CODE REVIEW

File: layers/+tools/realgud/packages.el [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+      - The symbol `local' directs Spacemacs to load the file at
+        `./local/PACKAGE/PACKAGE.el'
+
+      - A list beginning with the symbol `recipe' is a melpa
+        recipe.  See: https://github.com/milkypostman/melpa#recipe-format")
+
+(defun realgud/init-realgud ()
+  (use-package realgud
+    :defer t))
```
Comment (bmag) [inline layers/+tools/realgud/packages.el]: This is the correct way to install a package without configuring it. The problem is not in your code, might be a Spacemacs bug (see comment on the issue at realgud repo)

Comment (jluttine) [inline layers/+tools/realgud/packages.el]: Ok, thanks for the comment. Just for reference: https://github.com/rocky/emacs-dbgr/issues/77#issuecomment-215315686

File: layers/+tools/realgud/README.org [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#+TITLE: realgud layer
+
+# The maximum height of the logo should be 200 pixels.
+[[img/realgud.png]]
```
Comment (cpaulik) [inline layers/+tools/realgud/README.org]: If there is no logo, please remove this.

**Comment by rmhsilva** [comment]
Hi @jluttine, are you still intending to finish this?
Cheers

**Comment by jluttine** [comment]
@rmhsilva Not at the moment.. Perhaps at some point (if nobody else happens to provide the layer before that). I got stuck because I couldn't get it working and then I moved to other stuff.

**Comment by diadochos** [comment]
This works as intended as of today on my environment.

#### System Info :computer:
- OS: darwin
- Emacs: 24.5.50.1
- Spacemacs: 0.200.2
- Spacemacs branch: master (rev. 7bdde17)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(csv bibtex
     (auto-completion :variables auto-completion-enable-sort-by-usage t auto-completion-enable-snippets-in-popup t auto-completion-enable-help-tooltip t auto-completion-private-snippets-directory "~/.emacs.d.bak/snippets/")
     better-defaults ipython-notebook emacs-lisp
     (org :variables org-enable-github-support t)
     osx spell-checking syntax-checking
     (ruby :variables ruby-enable-enh-ruby-mode t ruby-version-manager 'rbenv ruby-test-runner 'rspec)
     python javascript latex yaml php gtags ess version-control git html clojure extra-langs dash docker my-keybindings my-config-from-spacemacs-base)
```

**Comment by skebanga** [comment]
I added this layer, and it installs fine, and I can launch it, but the keybindings don't seem to be working.

Are there any keybindings?

Other than add the package, is there any additional configuration required?

**Comment by bmag** [comment]
@skebanga this layer is a WIP (work in progress), so look at the code and figure it out. Right now, this layer just installs realgud and that's it.

@jluttine do you intend to resume work on this PR? I'd rather close it if it's been abandoned.

**Comment by jluttine** [comment]
@bmag Not at the moment. I'll close this for now, and if I ever find time to resume before anyone else makes this, I'll make a new pull request (or re-open this).

**Pull request closed without merging**